sycl/test-e2e/README.md

```diff
@@ -448,54 +448,23 @@ system.
-  processing sycl-ls output), or if its expected to always fail on run-time we
-  can use `XFAIL: run-mode`. This is because otherwise the test would compile
-  and pass on `build-only` mode and be reported as an `XPASS`.
-
-* To separate compilation and execution of tests, we classify `RUN:` directives
-as being either build or run lines. If a line contains `%{run}`,
-`%{run-unfiltered-devices}` or `%if run-mode` it is classified as a run line,
-otherwise it is classified as a build line.
-  * All `RUN:` lines that execute test binaries should be marked with either
-  `%{run}` or `%{run-unfiltered-devices}`. Otherwise they will be incorrectly
-  marked as a build line, likely causing a failure at the `build-only` stage as
-  we try to execute the program without having the appropriate devices.
-  * The vast majority of `RUN:` lines that do not execute the test binaries are
-  needed to either set up files prior to compilation, or to compile the binary,
-  as such `RUN:` lines are by default considered as build lines. In the case
-  that we need to run a line on the `run-only` system, and it does not make
-  sense to mark them with `%{run}` or `%{run-unfiltered-devices}`, we can mark a
-  line with `%if run-mode` to specifically make the line a run line. This
-  situation usually appears when we need to run a command in response to the
-  execution of the test binary.
-
-* Currently the `build-only` mode does not support logic to properly assess the
-features in `REQUIRES`/`UNSUPPORTED` to know if a test can be built in the
-system environment, or for `spir64`. Only tests that are marked with `REQUIRES:
-build-and-run-mode` or `UNSUPPORTED: true` are skipped. Thus if a test will fail
-building for the build environment we have on CI or for `spir64` we will need to
-mark this as `REQUIRES: build-and-run-mode`. This is only temporary solution,
-until further work is done to properly mark tests as unsupported on `build-only`
-based on features.
-
-* CPU and FPGA AOT tests are currently expected to fail when compiling and
-executing on separate machines. These failures occur on the `run-only` side,
-because during compilation the host machine's CPU architecture is targeted,
-which may be different than that of the running machine. These tests are marked
-as `REQUIRES: build-and-run-mode` as a result, until they can be refactored to
-compile for the architectures that will be used on the run side.
-
-#### Falling back to `full` testing mode on `run-only`
-
-To not lose coverage of tests marked as `REQUIRES: build-and-run-mode` when
-using `run-only` mode, lit can be called using
-`--param fallback-to-build-if-requires-build-and-run=True`. When this option is
-enabled in `run-only` mode, tests marked as requiring `build-and-run-mode` will
-fallback to running on `full` mode, instead of being reported as unsupported.
+  with `XFAIL` on a device specific feature. If it is expected to fail for any
```

Comment:
Refer to an expression instead of a single feature, similar to my other comment above.